ggu/main.c

```diff
@@ -726,6 +728,8 @@ run_ggu(struct net_config *net_conf,
 	pop_n_at_stage2(1);
 stage1:
 	pop_n_at_stage1(1);
+burst:
+	net_conf->back.total_pkt_burst -= ggu_conf->ggu_max_pkt_burst;
```

Comment:
Do we need to apply local variable here? As it's only the value from ``ggu_conf``.

Reply:
I think so. The idea is to avoid someone changing only in the `+=` assignment and forgetting the failure label.